portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/PagesConfiguration.path

```diff
@@ -52,7 +52,7 @@
 </tr>
 <tr>
 	<td>PAGES_ALL_PRIVATE_PAGES_CHECKBOX</td>
-	<td>//span[contains(.,'Private Pages')]//input[contains(@type,'checkbox')]</td>
+	<td>//span[contains(.,'Private Pages')]//input[contains(@type,'checkbox')] | //span[contains(.,'Staging Test Page')]//input[contains(@type,'checkbox')]</td>
```

Comment:
This path is supposed to be only for private pages. I was unable to find where is being used, but in theory, we should use another macro for a Staging page.